pkgs/development/libraries/quickfix/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool }:
+
+let
+  name = "quickfix";
+  version = "1.15.1";
+in stdenv.mkDerivation rec {
+  inherit name;
+
+  nativeBuildInputs = [ autoconf automake libtool ];
+
+  src = fetchFromGitHub {
+    owner = "${name}";
+    repo = "${name}";
```

```suggestion
    owner = pname;
```

You don't need `${}` if you are not interpolating the variable with a regular string